delfin/common/config.py

```diff
@@ -100,6 +100,16 @@
 CONF.register_opts(global_opts)
 
 
+security_opts = [
+    cfg.StrOpt('verify',
+               default=False,
+               help='False: Disable SSL certificate verification, '
+                    '/path/to/file: Use SSL certificate from file location')
+]
+
+CONF.register_opts(security_opts, group='security')
```

Comment:
it's confused, why create a group security? it's for all security?

Reply:
Modified to [storage_driver]